cluster/centos/deployAddons.sh

```diff
@@ -47,11 +47,11 @@ function deploy_dns {
 function deploy_dashboard {
   echo "Deploying Kubernetes Dashboard"
 
-  ${KUBECTL} apply -f ${KUBE_ROOT}/cluster/addons/dashboard/dashboard-secret.yaml
-  ${KUBECTL} apply -f ${KUBE_ROOT}/cluster/addons/dashboard/dashboard-configmap.yaml
-  ${KUBECTL} apply -f ${KUBE_ROOT}/cluster/addons/dashboard/dashboard-rbac.yaml
-  ${KUBECTL} apply -f ${KUBE_ROOT}/cluster/addons/dashboard/dashboard-controller.yaml
-  ${KUBECTL} apply -f ${KUBE_ROOT}/cluster/addons/dashboard/dashboard-service.yaml
+  ${KUBECTL} apply -f "${KUBE_ROOT}"/cluster/addons/dashboard/dashboard-secret.yaml
```

Comment:
Why about use a quoting style like: `"${RELEASES_DIR}/kubernetes/server/bin/kube-controller-manager"` and here only quote the variable?

Reply:
Ok, fixed in new commit